scripts/checkRequests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo
+echo "Starting time:"
+date
+source /opt/swefreq/venv/bin/activate
+export PYTHONPATH=/opt/swefreq/tornado
```

Comment:
The environment activation script is for getting the dependencies we need (peewee, tornado...), the PYTHONPATH is so that python find the ORM in `tornado/db.py`.